src/ReverseProxy/Management/ProxyConfigManager.cs

```diff
@@ -160,8 +168,18 @@ internal async Task<EndpointDataSource> InitialLoadAsync()
                 clusters.AddRange(config.Clusters ?? Array.Empty<ClusterConfig>());
             }
 
+            if (_configNotifier is not null)
+            {
+                _configNotifier.ConfigurationLoaded(ProxyConfigManager.ExtractListOfProxyConfigs(_configs));
+            }
```

Comment:
I think this accomplishes the same thing:
```suggestion
            _configNotifier?.ConfigurationLoaded(ProxyConfigManager.ExtractListOfProxyConfigs(_configs));
```

Reply:
@Tratcher Fixed